dev/bots/test.dart

```diff
@@ -706,25 +703,15 @@ Future<void> _runFlutterWebTest(String workingDirectory, List<String> tests) asy
 }
 
 Future<void> _pubRunTest(String workingDirectory, {
-  String testPath,
+  List<String> testPaths = const <String>[],
   bool enableFlutterToolAsserts = true,
-  bool useBuildRunner = false,
+  String coverage,
   bq.TabledataResourceApi tableData,
 }) async {
-  final List<String> args = <String>['run'];
-  if (useBuildRunner) {
-    final String posixTestPath = path.posix.joinAll(path.split(testPath));
-    args.addAll(<String>[
-      'build_runner',
-      'test',
-      '--build-filter=$posixTestPath/*.dill',
-      '--build-filter=$posixTestPath/**/*.dill',
-      '--',
-    ]);
-  } else {
-    args.add('test');
-  }
+  final List<String> args = <String>['run', 'test'];
   args.add(useFlutterTestFormatter ? '-rjson' : '-rcompact');
+  if (coverage != null)
+    args.add('--coverage=$coverage');
   int cpus;
```

Comment:
Cleanup: Hoist the `cpus` calculation and switch to the new list literal syntax.